src/System.Management.Automation/engine/hostifaces/PSTask.cs

```diff
@@ -35,16 +35,19 @@ internal sealed class PSTask : PSTaskBase
         /// <param name="scriptBlock">Script block to run in task.</param>
         /// <param name="usingValuesMap">Using values passed into script block.</param>
         /// <param name="dollarUnderbar">Dollar underbar variable value.</param>
+        /// <param name="currentLocation">Current working directory.</param>
         /// <param name="dataStreamWriter">Cmdlet data stream writer.</param>
         public PSTask(
             ScriptBlock scriptBlock,
             Dictionary<string, object> usingValuesMap,
             object dollarUnderbar,
+            PathInfo currentLocation,
```

Comment:
This looks like it modifies an existing public API, which would be a breaking change.

Would it be possible instead to add a new overload?

Reply:
It is not released.

Reply:
This is an experimental feature, subject to change.